kernel/tests/golden_tables.rs

```diff
@@ -129,10 +143,7 @@ fn assert_columns_match(actual: &[Arc<dyn Array>], expected: &[Arc<dyn Array>])
         let expected = normalize_col(expected.clone());
         // note that array equality includes data_type equality
         // See: https://arrow.apache.org/rust/arrow_data/equal/fn.equal.html
-        assert_eq!(
-            &actual, &expected,
-            "Column data didn't match. Got {actual:?}, expected {expected:?}"
-        );
+        assert_eq!(&actual, &expected, "Column data didn't match.");
```

Comment:
the assert macro already prints out left/right so we don't need the extra print in the message